dts/arm/st/u0/stm32u031X4.dtsi

```diff
@@ -27,3 +15,11 @@
 		};
 	};
 };
+
+&sram1 {
+	reg = <0x20000000 DT_SIZE_K(8)>;
+};
+
+&sram2 {
+	reg = <0x20080000 DT_SIZE_K(4)>;
+};
```

Comment:
In st .dtsi files, we generally don't overwrite nodes using `&node` references but directly "in place", so it would be:
```
/ {
	sram1: memory@20000000 {
		reg = <0x20000000 DT_SIZE_K(8)>;
	};
       [...]
```
I think, this should be kept for consistency and better readability of the .dtsi nodes (as it preserves "hierarchy": for instance we see that sram, is not part of node soc/, which is not the case using &node reference.).